### DISCUSSION TITLE
Language Concept in Engine Cockpit

### DISCUSSION THREAD

**Discussion by alexsuter** [axonivy/engine-cockpit#34] [Ideas]
I'll apply the language concept to the engine-cockpit.

User-Detail Page:
1) Add language and formatting language to user information card. (Later I want to have there two dropdown)
2) Email-Notification card will loos all language stuff and also accidently introduced mail notification process.

![image](https://user-images.githubusercontent.com/7498380/176422328-fe641219-2b5b-47a2-b687-a07aecb5896c.png)

Email-Service:
1) Here we loos the default language. Because the language is not attached to the email feature anymore.
![image](https://user-images.githubusercontent.com/7498380/176422620-b83600e0-2e96-4df9-bce1-b311ebb67266.png)

Security-System-Detail view:
1) I would introduce here two dropdowns to change the content language and formatting language.

![image](https://user-images.githubusercontent.com/7498380/176422777-4f6d4a01-5fce-4a8a-9387-84ef46553e2f.png)

Do you have better ideas or other opinions?

**Comment by alexsuter** [comment]
Hello @ivy-lli
CC: @ivy-cst 

You may want to share your opinion here?

**Reply by ivy-lli** [reply]
Sounds good to me!
As already once said, I still would vote to move the email settings to the security system. Maybe into the security detail view itself or at least to the navigation menu. But this is another discussion I think.

**Reply by alexsuter** [reply]
Ok. I didn't know that. Could be a good idea. But I won't do it now :)
